test/features/window.tab.manipulations.feature

```diff
@@ -15,7 +15,18 @@ Feature: Window/tab
     And I close current tab
 
   Scenario: "I close current tab" step should close recently opened tab
-    Given I open "http://localhost:9000"    
+    Given I open "http://localhost:9000"  
+    When I wait "5000" ms  
     When I open new window
     When I close current tab
     Then txtHeader from main page should be present
+
+  Scenario: "I switch to first tab" steps should open first tab
+    Given I open "http://localhost:9000"  
+    When I open new window
+    And I switch to first tab
+    Then txtHeader from main page should be present
+  
+  Scenario: "I switch to last tab" steps should open last tab
+    When I switch to last tab
```

Comment:
Each test should be independent from the others, so it looks like you have to add
```
Given I open "http://localhost:9000"  
And I open new window
And I switch to first tab
```
at the beginning of this test

Reply:
Implemented